Source/AliveLibAO/BaseAnimatedWithPhysicsGameObject.cpp

```diff
@@ -276,9 +321,20 @@ PSX_RECT* BaseAnimatedWithPhysicsGameObject::VGetBoundingRect_418120(PSX_RECT* p
     return pRect;
 }
 
-void BaseAnimatedWithPhysicsGameObject::SetTint_418750(const TintEntry* /*pTintArray*/, LevelIds /*levelId*/)
+void BaseAnimatedWithPhysicsGameObject::SetTint_418750(const TintEntry* pTintArray, LevelIds level_id)
 {
-    NOT_IMPLEMENTED();
+    while (pTintArray->field_0_level != static_cast<int>(level_id))
+    {
+        if (pTintArray->field_0_level == static_cast<int>(level_id) || pTintArray->field_0_level == static_cast<int>(LevelIds::eNone))
+        {
+            break;
```

Comment:
is this OG? If it is and the item isn't found it seems like it could crash :oddtism:

Reply:
yup OG